examples/Billing/AddBillingSetup.cs

```diff
@@ -172,38 +178,40 @@ public override string Description
-        /// to see if there are any approved billing setups. If there are any, the new billing setup
-        /// may have a starting date time of any day in the future. If not, the billing setup must
-        /// be set to start immediately.
+        /// Sets the starting and ending date times for the new billing setup. Queries the
+        /// customer's account to see if there are any approved billing setups. If there are any,
+        /// the new billing setup starting date time is set to one day after the last. If not, the
+        /// billing setup is set to start immediately. The ending date is set to one day after the
+        /// starting date time.
         /// </summary>
-        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
         /// <param name="googleAdsService">The Google Ads service client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
         /// <param name="billingSetup">The instance of BillingSetup whose starting date time will
         ///     be set.</param>
-        private void SetBillingSetupStartDateTime(long customerId, GoogleAdsServiceClient
-            googleAdsService, BillingSetup billingSetup)
+        private void SetBillingSetupStartDateTime(GoogleAdsServiceClient googleAdsService,
+            long customerId, BillingSetup billingSetup)
         {
-            // Query to see if there are any existing approved billing setups. See
-            // GetBillingSetup.cs for a more detailed example of requesting billing setup
+            // The query to search existing approved billing setups in the end date time descending
+            // order.
+            // See GetBillingSetup.cs for a more detailed example of requesting billing setup
             // information.
             string query = @"
                 SELECT billing_setup.end_date_time
                 FROM billing_setup
                 WHERE billing_setup.status = 'APPROVED'
                 ORDER BY billing_setup.end_date_time DESC";
 
+            // Issues a search request.
             PagedEnumerable<SearchGoogleAdsResponse, GoogleAdsRow> searchResponse =
                 googleAdsService.Search(customerId.ToString(), query);
 
             if (searchResponse.Any())
             {
-                // If there are any existing approved billing setups, the new billing setup can
-                // start immediately after the last.
+                // Retrieves the ending date time of the last billing setup.
                 string lastEndingDateTimeString = searchResponse.First().BillingSetup.EndDateTime;
 
-                // Check if the existing billing setup has no end date (i.e., is set to run
-                // indefinitely).
+                // If the ending date is null then it is set to run indefinitely. Creating a new
+                // billing setup can impact it.
```

Comment:
I believe "Creating a new billing setup can impact it" can be misleading. I suggest we remove it and add something like "Billing setups cannot overlap each other" in the conditional block below to better explain the issue.

Reply:
Agreed, it felt ambiguous.  Reworded.